src/game/AI/ScriptDevAI/scripts/world/item_scripts_wotlk.cpp

```diff
@@ -190,6 +190,45 @@ struct BlessingOfAncientKings : public AuraScript
     }
 };
 
+// 67799 - Mind Amplification Dish
+struct MindAmplificationDish : public SpellScript
+{
+    enum class MindAmplificationOutcomes : uint32
```

Comment:
If ur defining it within a class, enum class is overkill, you will just be writing more syntactic sugar everywhere imho and typing more. (ur alrdy not polluting global space cos its within that class)

Reply:
yeah I know, I'll remove it once I have a better idea about the probability distribution, thanks for the advice, tho

Reply:
actually you're absolute right there's no point in dragging this out, sorry.